config/oidc-auth.php

```diff
@@ -57,13 +57,18 @@
     | JWT claims in id_token required to authenticate. Arrays set required
     | elements in an array. Other values are matched exactly.
     |
-    | This can also be a Closure to check id_token. id_token will be passed as
-    | the first parameter. Return true to indicate a pass or false for a fail.
+    | You can provide a \Closure or
+    | \Laravel\SerializableClosure\SerializableClosure that accepts id_token as
```

Comment:
serialized SerializableClosure, not plain SerializableClosure

Reply:
fixed in new revision